mods/content/plant_dissection/plants.dm

```diff
@@ -0,0 +1,112 @@
+// TODO: unify with /obj/structure/flora/plant
```

Comment:
this is newer, right? i'd honestly rather merge `flora/plant` into `flora/growing` instead of the other way around. i guess the main difference is the icons anyway

Reply:
Main difference is that /growing doesn't have a seed datum associated, and we probably don't want /plant processing on a subsystem.